index.test.ts

```diff
@@ -138,40 +138,48 @@ function makeTest(args: string) {
 }
 
 test("simple", async () => {
-  const {stdout, stderr, exitCode} = await execa(script, [
+  const subprocess = spawn(process.execPath, [
+    script,
     "-C",
     "--githubapi", githubUrl,
     "--pypiapi", pypiUrl,
     "--registry", npmUrl,
     "-f", testFile,
   ]);
+  const {stdout, stderr} = await subprocess;
+  const {exitCode} = await subprocess.nodeChildProcess;
   expect(stderr).toEqual("");
   expect(stdout).toContain("prismjs");
   expect(stdout).toContain("https://github.com/silverwind/updates");
   expect(exitCode).toEqual(0);
 });
 
 test("empty", async () => {
-  const {stdout, stderr, exitCode} = await execa(script, [
+  const subprocess = spawn(process.execPath, [
+    script,
     "-C",
     "--githubapi", githubUrl,
     "--pypiapi", pypiUrl,
     "-f", emptyFile,
   ]);
+  const {stdout, stderr} = await subprocess;
+  const {exitCode} = await subprocess.nodeChildProcess;
   expect(stderr).toEqual("");
   expect(stdout).toContain("No dependencies");
   expect(exitCode).toEqual(0);
 });
 
 if (env.CI && !env.BUN) {
   test("global", async () => {
-    await execa("npm", ["i", "-g", "."]);
-    const {stdout, stderr, exitCode} = await execa("updates", [
+    await spawn("npm", ["i", "-g", "."]);
+    const subprocess = spawn("updates", [
       "-C",
       "--githubapi", githubUrl,
       "--pypiapi", pypiUrl,
       "-f", testFile,
     ]);
+    const {stdout, stderr} = await subprocess;
+    const {exitCode} = await subprocess.nodeChildProcess;
```

Comment:
Seems odd that there is no `subprocess.exitCode`, any insights why `nano-spawn` does not provide it?

Reply:
I see `nano-spawn` throws on errors and exit code is on https://github.com/sindresorhus/nano-spawn#subprocesserror, so I recommend just removing this check for exit code 0. It would throw in case of non-0 code, so the test would fail anyways.